tests/test_nest.py

```diff
@@ -98,6 +99,16 @@ def spy(_):
         self.assertGreaterEqual(err_mean, abs(mean_rate - expected_rate))
         self.assertGreaterEqual(err_var, var_rate - expected_var)
 
+    @patch("bsb_hdf5.connectivity_set.ConnectivitySet.get_local_chunks")
+    def test_empty_conn(self, get_content_mock):
+        # Override get_local_chunks to test empty connection sets
+        get_content_mock.return_value = []
+        cfg = get_test_config("gif_pop_psc_exp")
+        cfg.simulations["test_nest"].connection_models["gif_pop_psc_exp"].rule = None
+        network = Scaffold(cfg, self.storage)
+        network.compile()
+        network.run_simulation("test_nest")
```

Comment:
This test doesn't seem to assert anything? Could you add a comment perhaps that it is enough for this test not to fail?

PS: This would make it a "regression test", we could name it `test_regression_issue_XXX` and use the issue number of the problem that was reported.